relay-server/src/services/processor/ourlog.rs

```diff
@@ -22,8 +25,8 @@ use {
 };
 
 /// Removes logs from the envelope if the feature is not enabled.
-pub fn filter<Group>(
-    managed_envelope: &mut TypedEnvelope<Group>,
+pub fn filter<LogGroup>(
+    managed_envelope: &mut TypedEnvelope<LogGroup>,
```

Comment:
Sorry, my previous comment here was misleading, I think we can just drop the type parameter. This ensures that the function can only be called for actual logs.

```suggestion
pub fn filter(
    managed_envelope: &mut TypedEnvelope<LogGroup>,
```